### ISSUE TITLE
Restore unsaved changes after Restart

### ISSUE THREAD

**Issue by stefan123t** [linuxmint/xed#428]
```
 * Xed version (xed --version)
 * Distribution - (Mint 17.2, Arch, Fedora 25, etc...)
```

**Issue**
I sometimes loose open and unsaved changes in my xed tabs due to power outages, crashes, etc. 
I therefor have to work harder or remember to save early.
Autosave is only applicable for documents Named and saved once.
(see #353)

**Steps to reproduce**
Type something in an unsaved UnnamedX document.
Wait for autosave period to expire (1-10 min).
killall xed
Start xed again

**Expected behaviour**
Should have autosaved the changes during the last autosave period
Should reopen the unsaved UnnamedX document.
(see #226)

**Other information**
@JosephMcc explained in #353 (and unfortunately closed the issue) that xed does not auto-save Unnamed/Unsaved document changes to a common Temp folder. 
Along with #226 on restoring tabs, people (e.g. @dmitry) were asking for some auto-save functionality for Unsaved/Opened tabs (like e.g. in Notepad++ ) too.
With system failures in mind, having such a backup of unsaved files being restored when starting xed again would be appreciated.

I believe vim and others call it a swap file.
There should be a documented (or even menu driven ?) way of preventing the initial restore / loading of such auto-saved swap files, e.g. removing the respective files in ~/.xed/auto-save/ or some opened_tabs document with the paths.
This is to recover in case xed died due to too many opened tabs or some other issue and one does not want to restore / reopen all those tabs.

**Comment by dmitry** [comment]
My suggestion: backup data should be saved somewhere in the `~/.config/xed/backups/` or similar path, not along the other files in the directory the file you are working on, or most frequently I have an issues of a new notes. I'm using xed as a notepad, just to store quick to use information.

**Comment by stefan123t** [comment]
Thanks for your thumbs up. I got bitten once more by opening a huge file (2MB) in xed again.
Now my xed is unresponsive and I have no other resort as to restart the whole program, probably loosing all unsaved changes.
I wish such a Recover functionality would be there in my still favorite editor in Linux Mint. *sigh*

**Comment by dmitry** [comment]
@stefan123t yeah, I stopped to use xed, and now using libreoffice writer, it's overkill for most of the times, but at least you can always to restore the data. For the last ~5 years of using xed I've lost too much notes because of that.

**Comment by work-flower** [comment]
happy to hear improvement about that, shouldn't be that hard guys, go on :)

**Comment by slaygon** [comment]
I have a working version of this that I am running locally. Got tired of losing my notes, so it is what I run by default on my Mint. If you want to build it for yourself, you can find it at [autosave-session-restore](https://github.com/slaygon/xed/tree/autosave-session-restore).
